classes/runner.php

```
@@ -844,6 +844,27 @@ public function addExtension(extension $extension, extension\configuration $conf
 		return $this;
 	}
 
+	public function getExtension($classname)
+	{
+		$extensions = array();
+		foreach ($this->getExtensions() as $extension)
+		{
+			if (get_class($extension) != $classname)
+			{
+				continue;
+			}
+
+			$extensions[] = $extension;
+		}
+
+		if (1 != count($extensions))
+		{
+			throw new \InvalidArgumentException(sprintf("Extension '%s' not found", $classname));
```

please, can you throw the `mageekguy\atoum\exceptions\logic\invalidArgument`